bahmni-docker/README.md

```diff
@@ -63,6 +63,16 @@ docker exec -it {{PROXY_CONTAINER_NAME/ID}} sh -c \
 ``` 
 Restart the proxy container `docker restart {PROXY_CONTAINER_NAME/ID}}`
 
+## Setup for Macbook with M1 chip
+The below steps needs to be performed before docker-compose.
+1. Open `bahmni-proxy.conf`, update `Mutex default` to `Mutex posixsem`.
```

Comment:
Wouldn't Mutex posixsem work on other platforms ? If yes and it doesn't have any performance issues, then can we go with posixsem as default option so that the setup is similar across machines.

Reply:
have to test on other platforms

Reply:
@kalyan90 I have tried in my machine with Intel chip by updating `Mutex posixsem` and the container started properly and applications are accessible. So lets make it default, so that Intel and M1 chip users can directly start using the proxy image without local builds.

Reply:
Defaulted to `Mutex posixsem` in bahmni proxy